src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,120 @@
+package view;
+
+import static view.OutputMessages.DISPLAY_ABSENT_COUNT;
+import static view.OutputMessages.DISPLAY_ATTENDANCE_COUNT;
+import static view.OutputMessages.DISPLAY_ATTENDANCE_RECORD_PROMPT;
+import static view.OutputMessages.DISPLAY_ATTENDANCE_RECORD_RESULT;
+import static view.OutputMessages.DISPLAY_CHECK_ATTENDANCE_RESULT;
+import static view.OutputMessages.DISPLAY_HAS_PENALTY;
+import static view.OutputMessages.DISPLAY_LATE_COUNT;
+import static view.OutputMessages.DISPLAY_MODIFY_ATTENDANCE_RESULT;
+import static view.OutputMessages.DISPLAY_PENALTY_CREW;
+import static view.OutputMessages.DISPLAY_PENALTY_PROMPT;
+
+import domain.PenaltyStatus;
+import dto.AttendanceRecordResponse;
+import dto.CrewPenaltyResponse;
+import dto.ModifyAttendanceResponse;
+import dto.TotalRecordsResponse;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.TextStyle;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Locale;
+
+public class OutputView {
+    public static void displaySpacing() {
+        System.out.println();
+    }
+
+    public void displayPrompt() {
+        String koreanDayOfWeek = LocalDate.now().getDayOfWeek()
+                .getDisplayName(TextStyle.FULL, Locale.KOREAN);
+        System.out.println(
+                OutputMessages.DATE_PROMPT.format(LocalDate.now().getMonthValue(), LocalDate.now().getDayOfMonth(),
+                        koreanDayOfWeek));
+        System.out.println(OutputMessages.FIRST_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.SECOND_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.THIRD_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.FOURTH_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.QUIT_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.GUIDE_PROMPT.getFormat());
+    }
+
+    public void displayCheckAttendanceResult(AttendanceRecordResponse response) {
+        int month = response.date().getMonthValue();
+        int day = response.date().getDayOfMonth();
+        String koreanDayOfWeek = response.date().getDayOfWeek()
+                .getDisplayName(TextStyle.FULL, Locale.KOREAN);
+        String formattedTime = response.time().format(DateTimeFormatter.ofPattern("HH:mm"));
+        System.out.println(DISPLAY_CHECK_ATTENDANCE_RESULT.format(month, day, koreanDayOfWeek, formattedTime,
+                response.attendanceStatus().getMessage()));
+    }
```

Comment:
이 부분 개선해봅시다! 참고하면 좋을 부분 공유드려요

- https://tmd8633.tistory.com/12
- https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/

Reply:
감사합니다! 급하게 작성하다보니 공유주신 내용들을 놓치게 되었네요.

```suggestion
    public void displayCheckAttendanceResult(AttendanceRecordResponse response) {
        LocalDate date = response.date();
        int month = date.getMonthValue();
        int day = date.getDayOfMonth();
        DayOfWeek dayOfWeek = date.getDayOfWeek();
        String koreanDayOfWeek = dayOfWeek.getDisplayName(TextStyle.FULL, Locale.KOREAN);
        LocalTime time = response.time();
        String formattedTime = time.format(DateTimeFormatter.ofPattern("HH:mm"));
        System.out.println(DISPLAY_CHECK_ATTENDANCE_RESULT.format(month, day, koreanDayOfWeek, formattedTime,
                response.attendanceStatus().getMessage()));
    }
```

.의 깊이가 깊어지는 건 끊어서 해결해줄 수 있을 것 같고,
시간 값들은 미리 String으로 바꾸어두어 OutputView에서는 출력만 담당하는 것이 나을 수도 있을 것 같은데, 한 번 생각해보겠습니다.